website/content/docs/discovery/checks.mdx

```diff
@@ -255,7 +150,7 @@ Any output of the script is captured and made available in the
 `Output` field of checks included in HTTP API responses,
 as in this example from the [local service health endpoint](/api-docs/agent/service#by-name-json).
```

Comment:
hello! 👋🏻 the [link migration](https://github.com/hashicorp/consul/pull/15976) has been released so when you update with `main` this link should be updated to the new format, prefixed with `consul`. I'm going to run through the rest of the internal links here and make notes where updates should occur. Let me know if you have questions!